### DISCUSSION TITLE
Using with Ale, getting double diagnostics

### DISCUSSION THREAD

**Discussion by agis** [neoclide/coc.nvim#4678] [Q&A] [answered]
I'm using with Ale and have set `"diagnostic.displayByAle: true"`. However now I see the errors two times: one from Ale (the greyed comments) and one from coc (the popups with the red font):

![image](https://github.com/neoclide/coc.nvim/assets/827224/b8efae30-3141-422b-a90a-ca884ecd856d)

How can I disable the popups and keep only Ale's messages?

**Comment by agis** [comment, marked as answer]
Nevermind, I put "diagnostic.displayByAle" in the wrong place.

**Reply by sisrfeng** [reply]
Would you mind deleting this discussion? Since it provide no useful information.

**Reply by fannheyward** [reply]
What do you mean `no useful information`?

**Reply by sisrfeng** [reply]
It seems like a personal mistake, and I think the doc about `diagnostic.displayByAle` is clear enough.

**Reply by fannheyward** [reply]
Got your point, it's a usage issue, but I think we don't need to delete the discussion cause this makes clear the configuration needs to be placed the right place to work.
